Kwartracker/Scenes/Reusable/TextGroupView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  TextGroupView.swift
+//  Kwartracker
+//
+//  Created by Mickale Saturre on 5/31/21.
+//
+
+import SwiftUI
+
+struct TextGroupView: View {
+    let text: String
+    let fontSize: CGFloat?
+    let bottomMargin: CGFloat?
```

Comment:
```suggestion
    let bottomMargin: CGFloat
```

Reply:
<img width="510" alt="Screen Shot 2021-06-02 at 2 18 32 PM" src="https://user-images.githubusercontent.com/23580147/120433142-73884100-c3ad-11eb-8ee2-da6740c21736.png">
It will give me an error if I remove the "?"

Reply:
You can remove parameters `fontSize` and `bottomMargin`